meilisearch-auth/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 [dependencies]
 enum-iterator = "0.7.0"
 meilisearch-error = { path = "../meilisearch-error" }
-milli = { git = "https://github.com/meilisearch/milli.git", tag = "v0.25.0" }
+# milli = { path = "../../milli/milli" }
+milli = { git = "https://github.com/meilisearch/milli.git", branch = "nested_fields" }
```

Comment:
I conclude that you are awaiting the `nested_fields` PR to be merged on milli?

Reply:
I just reviewed https://github.com/meilisearch/milli/pull/458 for you and bors merged it.